Source/Core/Core/DolphinAnalytics.h

```diff
@@ -108,6 +109,7 @@ class DolphinAnalytics
 public:
   // Performs lazy-initialization of a singleton and returns the instance.
   static DolphinAnalytics& Instance();
+  ~DolphinAnalytics();
```

Comment:
[Rule of three](https://en.cppreference.com/w/cpp/language/rule_of_three.html) maybe?

```suggestion
  DolphinAnalytics(const DolphinAnalytics&) = delete;
  DolphinAnalytics& operator=(const DolphinAnalytics&) = delete;
  ~DolphinAnalytics();
```